src/Components/CommentsWidget.tsx

```diff
@@ -208,12 +208,37 @@ export function CommentForm(props: CommentFormProps) {
 
     const target = `${savedLocale}/${document.location.pathname.replace(/^\s*\/*\s*|\s*\/*\s*$/gm, '')}`;
 
+    function calculateMessageSusScore(message: string) {
+        let susScore = 0;
+        if (
+            ['@gmail.com', '@web.de', '@gmx.de', '@gmx.net', '@hotmail.com', '@me.com', '@mail.com'].filter((x) =>
```

Comment:
`.some()` would be better here—we don't care about which one matches, only whether any one does.